Client/mods/deathmatch/logic/CClientEntity.cpp

```diff
@@ -295,22 +293,22 @@ CLuaArgument* CClientEntity::GetCustomData(const char* szName, bool bInheritData
     // If none, try returning parent's custom data
     if (bInheritData && m_pParent)
     {
-        return m_pParent->GetCustomData(szName, true, pbIsSynced);
+        return m_pParent->GetCustomData(strName, true, pbIsSynced);
     }
 
     // None available
-    return NULL;
+    return {};
```

Comment:
Isn't `nullptr` better in such cases?

Reply:
{} is more universal and reliably. Just allow the compiler to deduce it, he knows better. This is also come in handy when a function's return argument is changed(say, from a pointer to bool). Then you don't need find and fix all places when the function return. This is a matter of a code style of course, but {} is preferable in generic C++.

Reply:
All your arguments are good for template/library code. 
> {} is more universal and reliably.

Specific instruments are always better and reliably. There is no reasons to make some parts of code more universal than they need to be.
> Just allow the compiler to deduce it, it knows better.

We write code for humans not the compiler. And we optimize code for programmers first. There is only one possible way to deduce return value in such cases. Does the compiler have some secrets? Or is there a reason to hide the value type from developers?

> This is also come in handy when a function's return argument is changed(say, from a pointer to bool). Then you don't need find and fix all places where the function returns.

This doesn't ever look like a significant time save. Developers waste more time when they read code. Additional context makes code easier to read.

> This is a matter of a code style of course, but {} is preferable in generic C++.

I checked UE5 and GTA5 code. UE5 uses `return {}` a lot ever for ints. GTA5 never uses this. [Stack overflow](https://stackoverflow.com/questions/39487065/what-does-return-statement-mean-in-c11) users prefer to use {} only for structs with a default initializer.

> This is a matter of a code style of course

It looks more like this. Do we want allow {} initialization for primitives to our code style?

Reply:
I feel like this is really just nitpicking. It isn't a major change in the style, so it's perfectly fine imo.